server/routes/boards.js

```diff
@@ -35,4 +36,9 @@ router.route('/cards').get(getCards, validateGetCards);
 router.route('/cards').post(createCard, validateCreateCard);
 router.route('/cards/move').put(moveCard, validateMoveCard);
 
+// Card Detail
+router.route('/cards/detail').get(protect,getDetails);
+router.route('/cards/createdetail').put(protect,createDetails);
```

Comment:
i would name these in camelCase format

```suggestion
router.route('/cards/createDetail').put(protect,createDetails);
```

or, you can actually just name it 

```suggestion
router.route('/cards/detail').put(protect,createDetails);
```

because the `put` method signals that it's a CREATE method already. same for other routes here

Reply:
Okay! sounds good.